Add links to soft navs experiment to SPA CWV guide and CrUX v RUM post (#9514)
GoogleChrome/web.dev: src/site/content/en/blog/vitals-spa-faq/index.md

  - philipwalton
  - yoavweiss
date: 2021-09-14

> updated: 2023-02-04

Author: ```suggestion
updated: 2023-02-10
```

limitations in the web platform), we are [actively working on closing those
gaps](#what-is-google-doing-to-ensure-mpas-do-not-have-an-unfair-advantage-compared-to-spas).

{% Aside %}

> Update: The work is now available for sites to experiment with. See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more details.

Author: ```suggestion
Update (February 2023): The work is now available for sites to experiment with. See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more details.
```

> Update: The work is anow available for sites to experiment with. See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more details.

Reviewer: ```suggestion
Update: The work is now available for sites to experiment with. See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more details.
```

> Update: The work is anow available for sites to experiment with. See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more details.

Reviewer: @malchata do you think we should also include the date that these updates were added, for additional context? For example "Update (February 2023):"

Reviewer 2: You could, but the `updated` value in the front matter has also been provided. I don't think it's necessary, but it wouldn't be a problem if it were added.

Author: Looks like it is used in a few other places, but no consistently. But since it does says Update, I'm gonna add it! Seems weird to have an "update" without saying when it was updated now I think about it.

migrating from an MPA to an SPA to actually compare the performance before and
after.

{% Aside %}

> Update: See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more information on a new "soft-navigations" API that is intended to allow better SPA measurement of the Core Web Vitals.

Author: ```suggestion
Update (February 2023): See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more information on a new "soft-navigations" API that is intended to allow better SPA measurement of the Core Web Vitals.
```

> Update (February 2023): See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more information on a new "soft-navigations" API that is intended to allow better SPA measurement of the Core Web Vitals.

Author: ```suggestion
Update (February 2023): See the [Experimenting with measuring soft navigations](https://developer.chrome.com/blog/soft-navigations-experiment/) post for more information on a new "soft-navigations" API that is intended to allow better Core Web Vitals measurements for SPAs.
```
